_posts/2020-03-30-crunch-on-arm.md

```diff
@@ -0,0 +1,84 @@
+
+Researchers have taken snapshots of the proteins of the COVID-19, called SARS-CoV-2, using techniques like X-ray crystallography and cryo-electron microscopy. But proteins don’t hold still. All the atoms in the protein and (its surroundings) are continually pushing and pulling on each other. These folding simulations help with modeling those physical interactions in the computer. These simulations reveal the different shapes a protein’s structure can take.
+
+Researchers usually look for a nice pocket on the surface of a protein where this little molecule that they design is inserted into a groove, but many proteins, particularly those in viruses, have seemingly smooth surfaces, making them hard to target.
+
+# Where does Distributive Computing like [Rosetta@Home](https://boinc.bakerlab.org/) and Folding@Home come into play?
+
+The study is usually carried out on very, very, very small timescales to capture the tiny jitters of atoms in proteins. Each step in the simulation is on the order of a femtosecond, or one quadrillionth of a second. To track protein motion over  a second, they’ve got to do a billion-squared operations on the computer, and each of those operations requires them to ask how every pair of atoms in the protein and surrounding solution are interacting with each other. By drawing on the computing power of many volunteers at once, distributive computing network performs calculations in a single month that could take an ordinary desktop computer 100 years.
+
+***
+
+# Arm and Rosetta@Home
+
+About a week ago, [David Tischler](https://community.arm.com/members/david-tischler) and [Rex St. John](https://www.rexstjohn.com/) contacted me to join what later became the "Rosetta on Arm Task Force" and help in porting and testing Rosetta on to the Arm architecture. I was also joined by members of the [The Baker Lab, University of Washington](https://www.bakerlab.org/) ie the creators and maintainers of the Rosetta@Home Project, and developers from [Neocortix](https://www.neocortix.com/). 
+
+The Idea was to get developers working in the arm ecosystem, who like many of us have mounds of arm development hardware sitting in their closet, to join Rosetta and donate spare CPU cycles to protein folding efforts. Now more than ever.
+
+While Dimitri(from Neocortix) worked on the initial patch that would allow cross-compilation on amd64 machines and an additional patch that allowed for docker builds, I worked on native aarch64 build and bare-metal testing. Sadly the source for Rosetta is not under any open-source license but available for academic use only. But what I can say is that the code was extremely portable requiring us to only add aarch64 as a target with some compiler flags in the build scripts, this was partly helped by the fact that amd64 was not the only target, ppc and ppc64 for mac are also available targets.
```

Comment:
Space after Dimitri
```suggestion
While Dimitri (from Neocortix) worked on the initial patch that would allow cross-compilation on amd64 machines and an additional patch that allowed for docker builds, I worked on native aarch64 build and bare-metal testing. Sadly the source for Rosetta is not under any open-source license but available for academic use only. But what I can say is that the code was extremely portable requiring us to only add aarch64 as a target with some compiler flags in the build scripts, this was partly helped by the fact that amd64 was not the only target, ppc and ppc64 for mac are also available targets.
```